packages/apps/public/locales/fr/translation.json

```diff
@@ -1357,7 +1357,7 @@
   "the stash account to nominate with": "le compte caché avec lequel désigner",
   "the supplied signature": "la signature fournie",
   "threshold": "seuil",
-  "tip": "pourboire",
+  "tip": "récompense",
```

Comment:
We should use one or the other everywhere, right now we have a mix, with "récompense" also being used for block reward. I'm not a fan of "pourboire" for tip either :man_shrugging: .

Reply:
Ok I'll leave that as is.